dataflow-migrate-schedules/README.adoc

```diff
@@ -0,0 +1,110 @@
+
+=== Running the maven command
+
+```
+./mvnw clean package
+```
+
+== Running The Project For Cloud Foundry
+
+=== Launching your migration
+1) Create a manifest.yml file in a work directory.
+```
+---
+applications:
+- name: schedulemigrator
+  host: schedulemigrator
+  memory: 1G
+  disk_quota: 1G
+  instances: 0
+  path: <location of the jar>
+  env:
+    SPRING_APPLICATION_NAME: schedulemigrator
+    spring_cloud_deployer_cloudfoundry_url: <CF API URL>
+    spring_cloud_deployer_cloudfoundry_org: <Your org>
+    spring_cloud_deployer_cloudfoundry_space: <your space>
+    spring_cloud_deployer_cloudfoundry_username: <Your CF user name>
+    spring_cloud_deployer_cloudfoundry_password: <Your CF password>
+    spring_cloud_deployer_cloudfoundry_skipSslValidation: <true/false>
+    spring_cloud_deployer_cloudfoundry_services: <your SCDF database service,your scheduler service>
+    spring_cloud_scheduler_cloudfoundry_schedulerUrl: <URL to scheduler service>
+    spring_profiles_active: cf
+    spring.cloud.deployer.cloudfoundry.healthCheckTimeout: 300
+    spring.cloud.deployer.cloudfoundry.apiTimeout: 300
+    dataflowServerUri: <Your SCDF server URI>
+    spring_cloud_task_closecontextEnabled: true
+    remoteRepositories_repo1_url: https://repo.spring.io/libs-snapshot
+  services:
+  - <Your SCDF database service>
+  - <Your SCDF scheduler service>
+```
+2) From the command line use the cf cli to log into your org and space for which you will migrate your schedules
+```
+cf login -a <your CF API endpoint>
+```
+-or if you need to skip ssl validation-
+```
+cf login -a <your CF API endpoint> --skip-ssl-validation
+```
+
+3) Now push the schedulemigrator from the directory where the manifest.yml is present:
+```
+cf push
+```
+
+3) To start the migration:
+From the `dataflow-migrate-schedules` directory launch the `runMigration.sh` using the commands below:
+```
+chmod +x scripts/runMigration.sh
+./scripts/runMigration.sh
+```
+=== Picking which schedules to migrate
+Use the `scheduleNamesToMigrate` property to specify a comma delimited list of
+the schedules you wish to migrate.  If you don't specify this property
+all schedules will be migrated.  For example:
+```
+./scripts/runMigration.sh --scheduleNamesToMigrate=task-job3,task-job1
+```
+
+=== Configuring Your Deployer Properties
+The following deployer properties will affect all schedules to be migrated.
+If a property is not set then the default will be used.
+
+==== Deployer properties to be applied to all migrated schedules:
+* healthCheckTimeout
+* apiTimeout
+* statusTimeout
+* stagingTimeout
+* startupTimeout
+* maximumConcurrentTasks
+* javaOpts
+
+NOTE: Descriptions of these properties can be found : https://github.com/cppwfs/spring-cloud-dataflow-samples/blob/SCDF-121/dataflow-migrate-schedules/src/main/java/io/spring/migrateschedule/service/MigrateProperties.java[here]
+
+=== Supported Databases
```

Comment:
Documentation should also note somewhere that we are not cleaning up the previously scheduled apps on CF.  That is the responsibility of the user if they are no longer needed.

Reply:
Good point.  Thanks.